workflow/testing/asserts.py

```diff
@@ -0,0 +1,76 @@
+from functools import partial
+import operator as op
+
+
+def assert_relation(r):
+    def _assert_relation(a, b, message=None):
+        assert r(a, b), f"{a} {_render(_inverse(r))} {b}" + ["", f": {message}"][message is not None]
+    return _assert_relation 
+
+def _not_contains(a, b):
+    return not(op.contains(a, b))
+
+def _inverse(r):
+    return {
+        op.is_: op.is_not,
+        op.is_not: op.is_,
+        op.eq: op.ne,
+        op.ne: op.eq,
+        op.lt: op.ge,
+        op.gt: op.lt,
+        op.le: op.gt,
+        op.ge: op.lt,
+        op.contains: _not_contains,
+        _not_contains: op.contains,
+    }[r]
+
+def _render(r):
+    return {
+        op.is_: 'is',
+        op.is_not: 'is not',
+        op.eq: '==',
+        op.ne: '!=',
+        op.lt: '<',
+        op.gt: '>',
+        op.le: '<=',
+        op.ge: '>=',
+        op.contains: '∋',
+        _not_contains: '∌',
+    }[r]
+
+
+class assert_:
```

Comment:
I don't like this myself, will move this stuff out to the module root instead
```
from workflow.testing import asserts
asserts.equal(1, 3)
```
is better

Reply:
Why do you prefer `assert.equal(1, 3)` over `assert(1 == 3)`

Reply:
I don't love this syntax:
```assert expression, 'my error message'```

I prefer
```
if not expression:
    raise AssertionError('my error message')
```

Reply:
@Jim-Holmstroem  Sorry I hijacked your comment by mistake (first wrote on mobile). Why is it not in the root right now? Did you run into problems?

Reply:
I hate operators as well
we could wrap that as well here :)
#anticorruptionlayer